src/script/ceph-backport.sh

```diff
@@ -140,13 +140,13 @@ else
 fi
 echo "Milestone is $milestone and milestone number is $milestone_number"
 
-if [ $(curl --silent http://tracker.ceph.com/issues/$issue.json | jq -r .issue.tracker.name) != "Backport" ]
+if [ $(curl --silent https://tracker.ceph.com/issues/$issue.json | jq -r .issue.tracker.name) != "Backport" ]
 then
     echo "http://tracker.ceph.com/issues/$issue is not a backport (edit and change tracker?)"
     exit 1
 fi
 
-title=$(curl --silent 'http://tracker.ceph.com/issues/'$issue.json?key=$redmine_key | jq .issue.subject | tr -d '\\"')
+title=$(curl --silent 'https://tracker.ceph.com/issues/'$issue.json?key=$redmine_key | jq .issue.subject | tr -d '\\"')
 echo "Issue title: $title"
 
 function prepare () {
```

Comment:
For some reason I can't comment on line 153.
You missed the following URL: `http://tracker.ceph.com/issues/$issue.json?include=relations`

Reply:
Added those as well